services/notifications/pkg/email/templates.go

```diff
@@ -33,6 +33,19 @@ var (
 		// ShareExpired email template, resolves via {{ .MessageBody }}
 		MessageBody: l10n.Template(`Your share to {ShareFolder} has expired at {ExpiredAt}
 
+Even though this share has been revoked you still might have access through other shares and/or space memberships.`),
+	}
+
+	ShareRemoved = MessageTemplate{
+		textTemplate: _textTemplate,
+		htmlTemplate: _htmlTemplate,
+		// ShareRemoved email template, Subject field (resolves directly)
+		Subject: l10n.Template(`{ShareSharer} unshared '{ShareFolder}' with you`),
+		// ShareRemoved email template, resolves via {{ .Greeting }}
+		Greeting: l10n.Template(`Hello {ShareGrantee},`),
+		// ShareRemoved email template, resolves via {{ .MessageBody }}
+		MessageBody: l10n.Template(`{ShareSharer} has unshared '{ShareFolder}' with you.
```

Comment:
@mmattel could you review these translation strings? They are going directly to transifex. Let's make sure we don't have to do another round like last time.

Reply:
Source looks good and is inline with the rest 👍